client/src/app/site/users/services/user-repository.service.ts

```diff
@@ -111,13 +111,15 @@ export class UserRepositoryService extends BaseRepository<ViewUser, User> {
     }
 
     /**
-     * Updates the password and sets the password without checking for the old one
+     * Updates the password, sets the current and 'default' password without
```

Comment:
Please redo this all. The default password should not be updated.

Reply:
There might be some conversation needed, because @emanuelschuetze explicitly requested that this function should reset the default password. I propose we make another issue/PR from this, as my part here is largely 'send request for resetting the default passwords' as tasked; and I do not know which concept and terms apply where.

Reply:
@FinnStutzenstein We should do it like in 2.3 that means the action generates a new personal password and saved it also as new default password - otherwise you can not see and send the new password to the user.
@MaximilianKrambach Please implement the 'generate password' function as discussed (like in 2.3 with update default password)!

Reply:
@emanuelschuetze You are right for multiselect. But changing this here will break the single-user-reset-password. I'll create an issue to make the behaviour clear after this PR
@MaximilianKrambach OK, go on. I'll create an issue.